sprint-3-report.qmd

```diff
@@ -0,0 +1,72 @@
+---
+title: Model Rollouts Summary
+
+crossref:
+  chapters: true
+---
+
+# At A Glance
+
+This page summarizes our findings and learnings from developing a cross-country poverty estimation model and rolling it out for 9 countries in Southeast Asia 
+
+
+
+## Model Rollout 
+
+The following figure describe our model rollout process for the 9 countries. 
+
+![Rollout Process showing the four main model rollout steps: generating tiles over country, filtering to populated tiles using High Resolution Settlement Layer (HRSL), feature engineering, and model estimation](assets/rollout_process.png)
+
+We were able to successfully use our model to estimate wealth over the 9 countries using this approach. The output wealth estimations captures macro-level urbanization, predicting relatively higher wealth at major urban centers and lower wealth at remote and sparsely populated areas. These results also show good agreement by Meta's RWI, showing that we can achieve good results in poverty estimation using openly-available datasets and methods. 
```

Comment:
"We were able to successfully use our model" change to "We were able to successfully train and use models".
Don't want to give the impression that we only have one model. 

"These results also show good agreement by Meta's RWI" -> "These results qualitatively show good agreement with Meta's RWI"

"poverty estimation" -> "wealth estimation"

"using openly-available datasets and methods" -> "using only openly-available datasets and <X> data points as training data from 4 countries"